packages/v4/src/content/design-guidelines/components/accordion/accordion.md

```diff
@@ -3,10 +3,34 @@ id: Accordion
 section: components
 ---
 
-An **accordion** is used to deliver a lot of content in a small space, allowing the user to expand and collapse the component to show or hide information.
+An **accordion** is an interactive container that expands and collapses to hide or reveal the nested content. It takes advantage of progressive disclosure to help reduce page scrolling by allowing users to choose to show or hide more detailed information as needed.
 
 ## Usage
+Accordions are one of many ways to handle large amounts of content. They are useful when you have a large amount of content, but limited space, with the header title giving an overview of the content that is hidden underneath it. Users can click or tap on the collapsed accordion to expand the content area and view additional details. Due to their high interaction cost, we recommend that you use them sparingly when you need an additional layer of structure to your content.
+
+## Single expand behavior
+This pattern is used to bring focus to a single row at the time. It is ideal for content that is mutually exclusive and does not need to be shown all at once. Whenever a new row is expanded, the previous one closes. Another benefit of this is real-estate management.
+
+### Use case
+Depending on the content, it can be useful to only expose users to one section at the time. This is particularly useful when working with forms. By default the first row is always open with all the others closed. Users can choose to close all rows, but only one can be opened at once.
+
+### Examples
+This variant is mostly used with navigation and sub-navigation.
+
+<img src="./img/single_expand_a.png" alt="Accordion single expand behavior" width="439"/>
+<img src="./img/single_expand_b.png" alt="Accordion single expand behavior" width="439"/>
```

Comment:
@kevinhatchoua Simply don't set a width on your images and they will take the full width of the content section of the page. They're resolution seems correct: https://patternfly-org-pr-2331-v4.surge.sh/v4/images/single_expand_a.1dc590b3bf3af5d251c03aaa153e5050.png

Reply:
All set.